requirements/base.in

```diff
@@ -5,6 +5,6 @@
 django-celery
 future
 celery>= 3.1.25,<4.0	
-Django >= 1.11
+Django >= 1.11, < 2.0
 django-model-utils==3.0.0	
-jsonfield
+jsonfield2==3.0.3
```

Comment:
Do we need to pin it?  Our default preference is to let `make upgrade` handle pinning to exact versions whenever possible.

Reply:
the latest version of `jsonfield2 3.1.0` drops the support for `python 3.5`

Reply:
Ok, then please move the version constraint to `requirements/constraints.txt` with that explanation in a comment.  Leave the dependency listing here as it was (or maybe add a comment at the end of the line explaining why we depend on it).